requirements-core.txt

```diff
@@ -24,3 +24,4 @@ pandas
 pyyaml
 openpyxl
 httpx~=0.14.0
+Cython==0.29.21
```

Comment:
This is unrelated. It probably came from something else you were doing. Can you remove it, please?

Reply:
Okay, I will remove it. I was trying to work with clear virtual environment and code would not run complaining about Cython. Perhaps it is a limitation of the virtualenv, I don't argue.